ArduCopter/RC_Channel.cpp

```diff
@@ -583,6 +574,13 @@ bool RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const AuxSwi
             }
             break;
 
+        case AUX_FUNC::ARMDISARM_WITH_AIRMODE:
```

Comment:
Why not just rename the virtual method instead?

Reply:
Both `ARMDISARM_WITH_AIRMODE` and `ARMDISARM` call the same function for all the other vehicles. I could have renamed the method and just have it local to copter I guess and call here. 

This highlights a issue that we did not consider when deciding not to change the behaviour. In keeping it the same on copter all other vehicles are now using `ARMDISARM_WITH_AIRMODE` despite that airmode on arming functionality only existing on copter. 

I guess plan C could be to do back to the change in behaviour but do the param conversion on copter.